metrics_layer/cli/seeding.py

```diff
@@ -304,12 +326,38 @@ def make_fields(self, column_data: str):
                 field["type"] = "time"
                 field["timeframes"] = ["raw", "date", "week", "month", "quarter", "year"]
                 field["datatype"] = metrics_layer_type
+            elif metrics_layer_type == "string" and auto_tag_searchable_fields:
+                field["field_type"] = "dimension"
+                field["type"] = "string"
+                column_cardinality_query = self.column_cardinality_query(
+                    column_name=row["COLUMN_NAME"], schema_name=schema_name, table_name=table_name
+                )
+                cardinality = self.run_query(query=column_cardinality_query)
+                cardinality = cardinality["CARDINALITY"].values[0]
+                if cardinality <= 100:
+                    field["searchable"] = True
             else:
                 field["field_type"] = "dimension"
                 field["type"] = metrics_layer_type
             fields.append(field)
         return fields
 
+    def column_cardinality_query(self, column_name: str, schema_name: str, table_name: str) -> str:
```

Comment:
This function implements the column cardinality query."

AFAIK, Postgres doesn't have an approximate count. We need to decide whether or not to disable for Postgres of keep it in, taking note of the performance/cost of count distinct over all dimensions.